information/gmt_data_server.txt

```diff
@@ -0,0 +1,59 @@
+# Information about all the remote data sets available on the GMT Data server
```

Comment:
If I understand it correctly, gmt_remote.c in GMT>=6.1 will read this file, then get the directory name, file name and other information. If that's true, we should use `server/earth_relief` instead of `data/server/earth_relief`, because the `data` directory is invisible to users.

Reply:
We could do that.  I put data in there so the file did not have an empty column for the symbolic links.  Suggestion? Put a period there?

Reply:
A period is OK. Or we could use `/` instead. 

`/` is the "root" of the data server, `/      earth_relief_01d.grd` will be mapped to https://oceania.generic-mapping-tools.org/earth_relief_01.grd.

Reply:
OK, I like the root /.